viogpu/common/baseobj.cpp

```diff
@@ -2,13 +2,13 @@
 #include "viogpu.h"
 
 _When_((PoolType & NonPagedPoolMustSucceed) != 0,
-    __drv_reportError("Must succeed pool allocations are forbidden. "
-        "Allocation failures cause a system crash"))
-    void* __cdecl operator new(size_t Size, POOL_TYPE PoolType)
+       __drv_reportError("Must succeed pool allocations are forbidden. "
```

Comment:
Looks more messed up now. But if this is the only thing, we can try to fix it later